app/pages/Identities/IdentityView.tsx

```diff
@@ -17,12 +17,9 @@ export default function IdentityView() {
 
     return (
         <>
-            <FailedIdentityModal
-                identityId={identity.id}
-                identityName={identity.name}
-                isRejected={identity.status === IdentityStatus.Rejected}
-                detail={identity.detail}
-            />
+            {isRejectedIdentity(identity) && (
```

Comment:
The `FailedIdentityModal` already does a check on whether `identity` is rejected. Do we need both checks?

Reply:
Not really, but I wanted no distinction between rejected and rejectedButWarned identities outside of `FailedIdentityModal`.
So this check is for the type checker, and so that the `FailedIdentityModal` logic only has to run for a failed identity.
The check inside is whether it is rejected or rejectedButWarned, which decides whether the modal should pop up.